src/altair/epoch_processing.rs

```diff
@@ -127,8 +212,14 @@ pub fn process_sync_committee_updates<
         MAX_VALIDATORS_PER_COMMITTEE,
         SYNC_COMMITTEE_SIZE,
     >,
-    _context: &Context,
+    context: &Context,
 ) -> Result<()> {
+    let next_sync_committee = mem::take(&mut state.next_sync_committee);
```

Comment:
so what happens if we are not aligned w/ the sync committee boundary?

`mem::take` here would just clear out the `next_sync_committee` which we don't want

hint: move this line inside the fn body